internal/web_server/web/handlers/api/organization_admin_api.go

```diff
@@ -0,0 +1,83 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/Molnes/Nyhetsjeger/internal/config"
+	"github.com/Molnes/Nyhetsjeger/internal/models/sessions"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users/access_control"
+	"github.com/Molnes/Nyhetsjeger/internal/utils"
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/dashboard_components/access_settings_components"
+	"github.com/labstack/echo/v4"
+)
+
+type OrganizationAdminApiHandler struct {
+	sharedData *config.SharedData
+}
+
+// Creates a new OrganizationAdminApiHandler
+func NewOrganizationAdminApiHandler(sharedData *config.SharedData) *OrganizationAdminApiHandler {
+	return &OrganizationAdminApiHandler{sharedData}
+}
+
+// Registers the organization admin related handlers to the given echo group
+func (oah *OrganizationAdminApiHandler) RegisterOrganizationAdminHandlers(g *echo.Group) {
+	g.POST("/access-control/admin", oah.postAddAdminByEmail)
+	g.DELETE("/access-control/admin", oah.deleteAdminByEmail)
+}
+
+// Handles a post request to add an admin by email. Email expected in form data.
+func (oah *OrganizationAdminApiHandler) postAddAdminByEmail(c echo.Context) error {
```

Comment:
`postAddAdminByEmail` could be `addAdminByEmail`. Which do we prefer?

Reply:
`post` signifies the expected http method. Can remove it tho